### ISSUE TITLE
Campagin notification is missing spacing above

### ISSUE THREAD

**Issue by clarkepaul** [silverstripe/silverstripe-framework#6039]
I can't access the campaign area at the moment due to a bug, so I need to wait until it is merged before fixing this spacing.
![image](https://cloud.githubusercontent.com/assets/555033/18658095/2ffd2998-7f54-11e6-9f04-8c568668b6f3.png)

Waiting on PR to be merged: https://github.com/silverstripe/silverstripe-framework/pull/6031

**Comment by clarkepaul** [comment]
Fixed by another PR
